travis.sh

```diff
@@ -8,7 +8,7 @@ function cleanPhpInMavenRepository {
 
 function configureTravis {
   mkdir ~/.local
-  curl -sSL https://github.com/SonarSource/travis-utils/tarball/v27 | tar zx --strip-components 1 -C ~/.local
+  curl -sSL https://github.com/SonarSource/travis-utils/tarball/v33 | tar zx --strip-components 1 -C ~/.local
```

Comment:
@ivandalbosco why this change? it should be additional line like this one https://github.com/SonarSource/sonar-javascript/blob/master/travis.sh#L12 (before line 20 of this file)